TypeScript/TreeMap/Getting-Started.md

```diff
@@ -267,3 +267,4 @@ The following screenshot displays the TreeMap when Labels are enabled.
 
 ![TypeScript TreeMap Label for Legend](Getting-Started_images/Getting-Started_img6.png)
 
+N> You can refer to our [JavaScript TreeMap](https://www.syncfusion.com/javascript-ui-controls/js-treemap) feature tour page for its groundbreaking feature representations. You can also explore our [JavaScript TreeMap](https://ej2.syncfusion.com/demos/#/bootstrap5/treemap/default.html) that shows you how to render and configure the TreeMap in JavaScript.
```

Comment:
Add example at the end of TreeMap
![image](https://github.com/syncfusion-content/typescript-docs/assets/95272441/877c4544-1830-43f7-adfa-4f1f495c4f69)

Reply:
resolved